### DISCUSSION TITLE
add  search organization  feature

### DISCUSSION THREAD

**Discussion by changchunhua2017** [dani-garcia/vaultwarden#2670] [Ideas]
i have too many organization （>50+），But currently only supports search project   
will consider support organization search feature ?

**Comment by BlackDex** [comment]
I suggest to read the following page https://bitwarden.com/help/searching-vault/

Also, since the 2022.5.x versions of the web-vault you can easily select each organization and only view those items on the top left side of the web-vault.

And, Vaultwarden is only the backend server. It does not maintain the clients then self.
